libs/openchallenges/ui/src/lib/organization-card/organization-card.component.ts

```diff
@@ -16,7 +16,7 @@ export class OrganizationCardComponent implements OnInit {
   mockStars!: number;
   mockMembers!: Avatar[];
   otherMembers!: number;
-  mockChallengesSupported!: number;
+  challengesSupported!: number | undefined;
```

Comment:
Is the exclamation point still required if `| undefined` is specified?

Reply:
It is not! Thanks for pointing it out.